evaluation/Input/Evaluation_plan.json

```diff
@@ -0,0 +1,257 @@
+﻿{
+    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.1/OSP_Qualification_Plan_Schema.json",
+    "Projects": [
+        {
+            "Id": "Sufentanil",
+            "Path": "https://github.com/Open-Systems-Pharmacology/Sufentanil-Model/releases/download/v1.1/Sufentanil.json",
```

Comment:
It's better to refer to the model snapshot by `"Path": "../../Sufentanil.json"`

Reply:
@Incei Please fix the path and the versions (s. above), then we can merge this PR

Reply:
@Incei ping